modules/foreman_domain.py

```diff
@@ -129,15 +129,10 @@ def main():
         supports_check_mode=True,
     )
 
-    (server_params, domain_dict, state) = module.parse_params()
+    (_, domain_dict, state) = module.parse_params()
```

Comment:
Do we still want to expose the server_params to the individual modules?

Reply:
Nope. I've just left it here for now as the other modules are still using it